include/mscclpp/proxy_channel.hpp

```diff
@@ -11,6 +11,8 @@
 
 namespace mscclpp {
 
+constexpr int MAX_INFLIGHT_REQUEST = 500;
```

Comment:
What's the default CQ size. It's a constant number or can be configured by users?

Reply:
The default value is 1024, depending on the hardware the value can be bigger, for example the Mellanox HCA can support up to 2048. So the user can configure a different value, but must be small than the hardware specification.

Reply:
> The default value is 1024, depending on the hardware the value can be bigger, for example the Mellanox HCA can support up to 2048. So the user can configure a different value, but must be small than the hardware specification.

Can we make this value configurable?

Reply:
Currently we are using the default value every time that we create IB connection at the executor:
![image](https://github.com/user-attachments/assets/26b294f7-85aa-4a35-b4fc-8f16415e094d)

Maybe we could use some env variable to allow the user change it.